.github/workflows/build.yml

```
@@ -254,20 +261,28 @@ jobs:
+          MERGE_SHA: ${{ needs.setup.outputs.latestMergeSha }}
+          BUILD_TIMESTAMP: ${{ needs.setup.outputs.timeStamp }}
 
       #Publish to GitHub Packages.
       - name: Publish tar.gz package to GPM
         run: |
-          ./mvnw -B -pl liquibase-dist clean deploy -DskipTests=true "-Dbuild.timestamp=${{ needs.setup.outputs.timestamp }}" "-Dbuild.repository.owner=liquibase" "-Dbuild.repository.name=liquibase" "-Dbuild.branch=${{ needs.setup.outputs.thisBranchName }}" "-Dbuild.number=${{ github.run_number }}" "-Dbuild.commit=${{ needs.setup.outputs.thisSha }}"
+          ./mvnw -B -pl liquibase-dist clean deploy -DskipTests=true "-Dbuild.timestamp=$BUILD_TIMESTAMP" "-Dbuild.repository.owner=liquibase" "-Dbuild.repository.name=liquibase" "-Dbuild.branch=$BRANCH_NAME" "-Dbuild.number=${{ github.run_number }}" "-Dbuild.commit=$MERGE_SHA"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH_NAME: ${{ needs.setup.outputs.thisBranchName }}
+          MERGE_SHA: ${{ needs.setup.outputs.thisSha }}
```

actionlint (1.7.12)

[error] property "thissha" is not defined in object type {latestmergesha: string; thisbranchname: string; timestamp: string; unmodifiedbranchname: string}

(expression)